core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -74,8 +75,10 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
         internetAccessible => {
           if (internetAccessible) {
             this.enableRTEicons();
+            this.connectedToInternet = true;
           } else {
             this.disableRTEicons();
+            this.connectedToInternet = false;
```

Comment:
this.connectedToInternet = internetAccessible;

Reply:
Done.